data/seeds/009_courses.js

```diff
@@ -18,6 +18,7 @@ exports.seed = function (knex) {
       number_of_sessions: 4,
       difficulty: 'Easy',
       session_type: 'Group',
+      syllabus_link: 'https://docs.google.com',
```

Comment:
Does this force syllabuses to be on Google docs? Is there a way to make the link more universal, or is it not necessary?